### PR TITLE
Add init subcommand to ab

### LINKED ISSUE #61: introduce init command
The command will add a template playbook with all the vars.

Edit: the expectation here is that a user would run `ansible-bender init` in the root of a git project. The command would create a new yaml file - an ansible playbook. The playbook would look like this:
```yaml
---
- name: Containerized version of $project
  hosts: all
  vars:
    a_variable: value
    # configuration specific for ansible-bender
    ansible_bender:
      base_image: fedora:latest
      target_image:
        # command to run by default when invoking the container
        cmd: /command.sh
        name: $project
      working_container:
        volumes:
        # mount this git repo to the working container at /src
        - "{{ playbook_dir }}:/src"
  tasks:
  - name: install dependencies needed to run project $project
    package:
      name:
      - a_package
      - another_package
      state: present
```

After the file is created on disk, ab should inform the user what happened and what are the next steps and how to run a build.

### PR DESCRIPTION
**Pull request by kmehant**
Fixes #61

### CODE REVIEW

**Comment by kmehant** [comment]
@TomasTomecek Please review!

File: ansible_bender/utils.py [hunk 1]
```diff
@@ -17,6 +17,19 @@
 out_logger = logging.getLogger(OUT_LOGGER)
 
 
+# template playbook dictionary
+playbook_template = {'hosts': 'all',
```
Comment (TomasTomecek) [inline ansible_bender/utils.py]: I'd actually store this as a string with yaml inside so we can precisly control the formatting -- I'm pretty sure that yaml.safe_dump will make it hard to edit afterwards.

Comment (TomasTomecek) [inline ansible_bender/utils.py]: and store it in constants please

File: tests/functional/test_cli.py [hunk 2]
```diff
@@ -70,3 +70,9 @@ def test_get_logs(target_image, tmpdir):
     assert "TASK [Gathering Facts]" in out
     assert "failed=0" in out
     assert "TASK [print local env vars]" in out
+
+def test_init(tmpdir):
+    cmd = ["init"]
+    ab(cmd, str(tmpdir))
+    out = ab(["init"], str(tmpdir), return_output=True)
+    assert out.startswith("Created a playbook template as template_playbook.yml")
```
Comment (TomasTomecek) [inline tests/functional/test_cli.py]: also please try to load the file using yaml to make sure it's correct (after the change I described above)

File: ansible_bender/cli.py [hunk 3]
```diff
@@ -347,6 +355,12 @@ def _push(self):
         # force = self.args.force
         self.app.push(target, build_id=build_id, force=False)
 
+    def _init(self):
+        with open('template_playbook.yml', 'w') as fd:
```
Comment (TomasTomecek) [inline ansible_bender/cli.py]: how about we call it just playbook?

File: ansible_bender/cli.py [hunk 4]
```diff
@@ -347,6 +355,12 @@ def _push(self):
...
+            yaml.safe_dump(playbook_template, fd, indent=2, default_flow_style=False)
+        print("Created a playbook template as template_playbook.yml")
```
Comment (TomasTomecek) [inline ansible_bender/cli.py]:
```suggestion
        print("Created an Ansible playbook template as template_playbook.yml")
```

File: ansible_bender/cli.py [hunk 5]
```diff
@@ -347,6 +355,12 @@ def _push(self):
...
+        print("Edit the playbook accordingly and run the build subcommand")
```
Comment (TomasTomecek) [inline ansible_bender/cli.py]:
```suggestion
        print("Edit the playbook accordingly and run the build subcommand:\n\nansible-bender build playbook.yaml")
```

**Comment by TomasTomecek** [comment]
also please rebase since I merged your clean PR

**Comment by kmehant** [comment]
@TomasTomecek not sure why the builds are failing as I could not find tests failing related to this PR 🤔

**Comment by kmehant** [comment]
@TomasTomecek All checks passed 🎉 shall we merge this now?

**Pull request merged**